umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/related_url.clj

```diff
@@ -17,6 +17,17 @@
       :Type "GET RELATED VISUALIZATION"})
    (values-at multimedia-sample "URL")))
 
+(defn- isOpenSearchGetCapabilities?
+  "Checks to see if the passed in url-types and mime-type make the related URL
+   and open search URL that should be translated from a USE SERVICE API type to the new
+   GET CAPABILITIES type."
+  [url-types mime-type]
+  (let [subtype (:Subtype url-types)]
+    (def subtype subtype)
+    (and (and subtype mime-type)
```

Comment:
Why the double `and`?

Reply:
Is it better to write it like the following?
(and subtype
         mime-type
         (= "opensearch" (string/lower-case subtype))
         (= "application/opensearchdescription+xml" mime-type))